src/components/feedback/Popover.tsx

```diff
@@ -133,9 +133,9 @@ function usePopoverPositioning(
 
     // First of all, open popover if it's using the native API, otherwise its
     // size is 0x0 and positioning calculations won't work.
-    const popover = popoverRef.current;
+    const popover = popoverRef.current!;
```

Comment:
Since this is a local hook I think it is OK to rely on the caller to ensure that `popoverRef.current` is set if `popoverOpen` is true. If this hook was exposed for consumers of the crate to use, I think we'd need to be more defensive.